packages/components/data-table/src/data-table.tsx

```diff
@@ -20,6 +21,7 @@ import HeaderCell from './header-cell';
 import DataRow from './data-row';
 import useManualColumnResizing from './use-manual-column-resizing-reducer';
 import ColumnResizingContext from './column-resizing-context';
+import { DataTableManagerContext } from '@commercetools-uikit/data-table-manager/src/data-table-manager';
```

Comment:
We should not import from internals of a package but just from what they export.
Importing from internals is fragile as we would be depending on its current implementation and will fail if it is refactored.

Reply:
updated here: https://github.com/commercetools/ui-kit/pull/2807/commits/e02436f523fc54b93c131ade1cb3b5b26bb4a20ahttps://github.com/commercetools/ui-kit/pull/2807/commits/e02436f523fc54b93c131ade1cb3b5b26bb4a20a